docs/platforms/javascript/common/install/esm.mdx

```diff
@@ -10,20 +10,19 @@ supported:
   - javascript.koa
 ---
 
-When running your application in ESM mode, you can't use `require()` to load modules. Instead, you have to use the  `--import` command line options to load a module before the application starts:
-
-Adjust the Node.js call for your application to use the [--import](https://nodejs.org/api/cli.html#--importmodule) parameter and point it at `instrument.js`, which contains your `Sentry.init()` code:
+When running your application in ESM mode, you need to pass an [--import](https://nodejs.org/api/cli.html#--importmodule) command line option to register the SDK's ESM module loader so that it can properly instrument libraries.
 
 ```bash
-# Note: This is only available for Node v18.19.0 onwards.
-node --import ./instrument.mjs app.mjs
+node --import @sentry/node/import app.mjs
 ```
 
-We do not support ESM in Node versions before 18.19.0.
+<Alert level="warning">
+  We do not support ESM in Node versions before 18.19.0.
+</Alert>
 
 ## When to use this
```

Comment:
```suggestion
```
I think this whole section makes more sense as an introduction at the top of the page.